kong/tools/timestamp.lua

```diff
@@ -5,56 +5,71 @@
 -- @module kong.tools.timestamp
 
 local luatz = require "luatz"
-local _M = {}
+local tz_time = luatz.time
+local tt_from_timestamp = luatz.timetable.new_from_timestamp
+local tt = luatz.timetable.new
+local math_floor = math.floor
 
 --- Current UTC time
--- @return UTC time
-function _M.get_utc()
-  return math.floor(luatz.time()) * 1000
+-- @return UTC time in milliseconds since epoch, but with SECOND precision.
+local function get_utc()
+  return math_floor(tz_time()) * 1000
```

Comment:
I don't know the reasons to specifically use the `luatz` library, hence didn't change that